docs/src/operations/requirements.md

```diff
@@ -55,13 +63,17 @@ made.
 - CPU
   - 16 cores / 32 threads, or more
 - RAM
-  - 512 GB or more if `account-index` is used
+  - 512 GB or more if an `account-index` is used, 1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
 - Disk
   - Consider a larger ledger disk if longer transaction history is required
   - Accounts and ledger should not be stored on the same disk
 
 ## Virtual machines on Cloud Platforms
 
+**NOTE**: Running on cloud platforms is **not recommended** (especially on
+mainnet) due to excessive egress costs and bad co-tenants starving your node of
```

Comment:
```suggestion
mainnet-beta) due to excessive egress costs and bad co-tenants starving your node of
```

Reply:
Fixed in cf8d6dbe7891746c9b46d8f173d9424687961d72